compiler/ztests/pushdown.yaml

```
@@ -4,11 +4,11 @@ script: |
   echo === distinct
   super compile -C -O 'from file | distinct a | yield b'
   echo === fork-join
-  super compile -C -O 'fork (=> from file1 => from file2) | join on a b | yield c'
+  super compile -C -O 'fork (=> from file1 => from file2) | join on right.b=left.a | yield left.c,right.d'
```

I'd like to keep the fields in alphabetical order for readability.

```suggestion
  super compile -C -O 'fork (=> from file1 => from file2) | join on right.a=left.b | yield left.c,right.d'
```